tests/regression/xenforo-plugin/9511100.yaml

```
@@ -13,11 +13,12 @@ tests:
             dest_addr: 127.0.0.1
             headers:
               Host: localhost
-              User-Agent: OWASP CRS
+              User-Agent: "OWASP CRS test agent"
               Accept: text/xml,application/xml,application/xhtml+xml,text/html;q=0.9,text/plain;q=0.8,image/png,*/*;q=0.5
             port: 80
             method: GET
-            uri: /proxy.php
+            version: HTTP/1.1
+            uri: /get/proxy.php
             data: |
               referrer=something
```

``Something`` won't trigger that rule, add an example domain name so we can test if that rule is actually being disabled correctly
```suggestion
              referrer=https://example.com
```